Dockerfile

```diff
@@ -1,24 +1,25 @@
-FROM golang:1.18-alpine3.16 AS build
+FROM golang:1.23-alpine3.20 AS build
 
-ARG consul_version=1.12.3
+ARG consul_version=1.19.2
 ADD https://releases.hashicorp.com/consul/${consul_version}/consul_${consul_version}_linux_amd64.zip /usr/local/bin
-RUN cd /usr/local/bin && unzip consul_${consul_version}_linux_amd64.zip
+RUN cd /usr/local/bin && unzip -o consul_${consul_version}_linux_amd64.zip
 
-ARG vault_version=1.11.0
+ARG vault_version=1.17.5
 ADD https://releases.hashicorp.com/vault/${vault_version}/vault_${vault_version}_linux_amd64.zip /usr/local/bin
-RUN cd /usr/local/bin && unzip vault_${vault_version}_linux_amd64.zip
+RUN cd /usr/local/bin && unzip -o vault_${vault_version}_linux_amd64.zip
 
-RUN apk update && apk add --no-cache git
+RUN apk add --no-cache git libcap
```

Comment:
Why add `libcap` package if you are not calling `setcap` or binding to any privileged ports?

Reply:
Hi, the `setcap` call got removed. Sorry. Fixed.